Drawer menu on desktop

KyleJu opened the pull request:
#3014.

jrobbins commented on client-src/elements/chromedash-drawer.js:
```diff
@@ -0,0 +1,165 @@
+        nav [active] a {
+          color: var(--nav-link-active-color);
+        }
+        nav .nav-dropdown-container {
```
Maybe we could just make the Stats item always open rather than collapsable.  We have enough space and it would simplify the code and the UX if it was just always open.

KyleJu replied:
Done

jrobbins commented on client-src/elements/chromedash-drawer.js:
```diff
@@ -0,0 +1,165 @@
+
+  renderTabs() {
+    return html`
+      <sl-drawer label="Menu" placement="start" open=true class="drawer-placement-start" contained>
```
Can we hide the "X" icon and the whole row that says "Menu" by specifying attribute "no header"?  The user can already toggle the menu closed by clicking the hamburger icon again.

KyleJu replied:
done

jrobbins reviewed (changes requested):
This looks very good so far. 

Remember to update the TODOs listed in the PR description.

jrobbins commented:
Also, PR title says "on desktop" but you did incorporate support for mobile into the same PR, so the title should be updated.

KyleJu commented:
> Also, PR title says "on desktop" but you did incorporate support for mobile into the same PR, so the title should be updated.

Addressed all comments and TODOs. It is easier if this is reviewed commit by commit.

The only caveat is that the old `chromedash-featurelist` does not shift like other pages. I tried a few things programmatically but felt like I spent enough time making it work . Maybe it is time to use the new feature page?

jrobbins reviewed (approved):
Non-shifting old all-features page is OK for now.

KyleJu commented:
There are a few more issues with chromedash-featurelist
- my feature is shown when user isn't log=inn
- shifting isn't working
- sign-in icon does not render properly

Will send out a follow-up PR for them
